networkoperator/v1/001-egressrouter-crd.yaml-patch

```diff
@@ -0,0 +1,40 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/addresses/properties/ip/anyOf
+  value: 
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/addresses/properties/gateway/anyOf
+  value: 
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/redirect/anyOf
+  value: 
+    - properties:
+        fallbackIP:
+          format: ipv4
+    - properties:
+        fallbackIP:
+          format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/redirect/properties/redirectRules/anyOf
+  value: 
+    - properties:
+        destinationIP:
+          format: ipv4
+    - properties:
+        destinationIP:
+          format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/minProperties
+  value: 4
```

Comment:
why these?

Reply:
please check https://coreos.slack.com/archives/CDCP2LA9L/p1613654960279300 and https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.5.1. It is a way to restrict the _number of properties_.

Reply:
I know what it means. Having it here is contradicting API conventions. We have pruning. Specifying unknown fields is valid, but they will get pruned.

Reply:
removed them and replaced them with the idea you gave.